working-groups.md

```diff
@@ -142,14 +143,12 @@ repository that need to expose documentation.
 | Slack Channel              | [#docs](https://tektoncd.slack.com/messages/docs)                                                                                                                                                                                            |
 
 
-| &nbsp;                                                     | Facilitators      | Company | Profile                                     |
-| --------------------------------------------------------   | ----------        | ------- | ---------------------------------------     |
-| <img width="30px" src="https://github.com/kimsterv.png">   | Kim Lewandowski   | Google  | [kimsterv](https://github.com/kimsterv)     |
-| <img width="30px" src="https://github.com/spomorski.png">  | Serge Pomorski    | Google  | [spomorski](https://github.com/spomorski)   |
-| <img width="30px" src="https://github.com/vdemeester.png"> | Vincent Demeester | Red Hat | [vdemeester](https://github.com/vdemeester) |
-| <img width="30px" src="https://github.com/anneqm.png"> | Anne Michalak | IBM | [anneqm](https://github.com/anneqm) |
-| <img width="30px" src="https://github.com/popcor255.png"> | Napoleon Santana | IBM | [popcor255](https://github.com/popcor255) |
-| <img width="30px" src="https://github.com/AlanGreene.png">    | Alan Greene     | IBM     | [AlanGreene](https://github.com/AlanGreene)
+| &nbsp;                                                     | Facilitators     | Company | Profile                                     |
+| --------------------------------------------------------   | ----------       | ------- | ---------------------------------------     |
+| <img width="30px" src="https://github.com/spomorski.png">  | Serge Pomorski   | Google  | [spomorski](https://github.com/spomorski)   |
```

Comment:
looks like this was already broken (probably i did this XD), https://github.com/spomorski isnt right, i think this should be https://github.com/sergetron

Reply:
:+1: